pg_search/concurrently.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/pg_search/src/bootstrap/create_bm25.rs b/pg_search/src/bootstrap/create_bm25.rs
```

Comment:
What's this file?

Reply:
Sorry, pls ignore this file. As i mentioned in https://github.com/paradedb/paradedb/issues/1744#issuecomment-2397838266, it looks like the CONCURRENTLY option is not supported via executing either from procedure or a function.

Reply:
Hmm. Yet another reason to move to the standard `CREATE INDEX` command